common/src/main/java/com/wynntils/features/user/TooltipScaleFeature.java

```diff
@@ -10,10 +10,13 @@
 import com.wynntils.core.features.properties.FeatureInfo;
 import com.wynntils.core.features.properties.FeatureInfo.Stability;
 import com.wynntils.mc.event.ItemTooltipRenderEvent;
-import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.mc.utils.ComponentUtils;
 import com.wynntils.mc.utils.McUtils;
 import com.wynntils.wc.utils.WynnUtils;
+import java.util.Collections;
+import java.util.List;
 import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @FeatureInfo(stability = Stability.STABLE, category = "Item Tooltips")
```

Comment:
I just realized the name of the class, `TooltipScaleFeature`, is not as spot-on anymore, now that we do both scaling and wrapping. I don't have any good suggestion (`TooltipFittingFeature`???), but if you figure out a good name that captures both scaling and wrapping, please feel free to rename the class. :) (If you do, remember to update the i18n resources.)

Reply:
I ended up using TooltipFittingFeature, I think it's the best term to summarize what it's doing.